server/database/models/book.js

```diff
@@ -2,10 +2,9 @@ export default (sequelize, DataTypes) => {
   const Book = sequelize.define('Book', {
     title: DataTypes.STRING,
     description: DataTypes.TEXT,
-    isbn: DataTypes.TEXT,
+    isbn: DataTypes.BIGINT,
```

Comment:
ISBN not BIGINT

Reply:
Biola also

Reply:
kindly revert back so biola can create a bug fix for this or you can assist him